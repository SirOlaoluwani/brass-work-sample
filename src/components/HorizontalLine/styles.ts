import { StyleSheet } from "react-native";

export const styles = StyleSheet.create({
  container: {
    height: 1,
    backgroundColor: "#9e9e9e47",
  },
});
