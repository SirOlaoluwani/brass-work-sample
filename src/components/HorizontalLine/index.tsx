import React from "react";
import { View } from "react-native";
import { styles } from "./styles";

const HorizontalLine = () => {
  return <View style={styles.container} />;
};

export default HorizontalLine;
