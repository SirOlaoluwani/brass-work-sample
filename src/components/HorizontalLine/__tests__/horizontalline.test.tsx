import React from "react";
import renderer from "react-test-renderer";

import HorizontalLine from "..";

describe("<HorizontalLine />", () => {
  it("matches the snapshot", () => {
    const tree = renderer.create(<HorizontalLine />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
