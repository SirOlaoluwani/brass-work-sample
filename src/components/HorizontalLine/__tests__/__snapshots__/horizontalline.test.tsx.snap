// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HorizontalLine /> matches the snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#9e9e9e47",
      "height": 1,
    }
  }
/>
`;
